src/keys.cc

```diff
@@ -229,6 +229,10 @@ UnitTest test_keys{[]()
          alt('j'),
          ctrl('r'),
          shift(Key::Up),
+         ctrl('['),
+         ctrl('\\'),
```

Comment:
This is here just to double-check that no accidental escaping shenanigans would break people's ability to map `<c-\>`.